apps/sms/index.html

```diff
@@ -134,7 +134,7 @@
   <body role="application" class="skin-comms theme-communications">
     <iframe src="/views/shared/shim_host.html" class="shim-host" hidden></iframe>
     <article id="main-wrapper" class="wrapper">
-      <section class="panel panel-InboxView" role="region" aria-hidden="true">
+      <section class="panel panel-InboxView panel-hidden" role="region" aria-hidden="true">
```

Comment:
`.panel-hidden` is display: none actually that make sure we do nothing for the panels which are not necessary to be displayed first.